foosball/static_src/js/foosball.js

```diff
@@ -0,0 +1,14 @@
+$(document).ready(function() {
```

Comment:
`$` protected due to JS insanity and moved the code to a more appropriate place, good point.